SimpleServlet/src/classes/User.java

```diff
@@ -0,0 +1,22 @@
+package classes;
+
+public class User {
+
+	public int id;
```

Comment:
Make this field and next three private and create get/set method where it need. Explanation: for example, take the id field, it is generated automatically by mongodb. If another class will have direct access to this field, it is likely that during the program will be given an incorrect value, such as a negative number. Read more info about Encapsulation: https://stackify.com/oop-concept-for-beginners-what-is-encapsulation/